packages/expo-font/ios/EXFont/EXFontLoaderProcessor.h

```diff
@@ -2,7 +2,13 @@
 
 #import <Foundation/Foundation.h>
 #import <UMFontInterface/UMFontProcessorInterface.h>
+#import <EXFont/EXFontManager.h>
 
 @interface EXFontLoaderProcessor : NSObject <UMFontProcessorInterface>
 
+- (instancetype)initWithFontFamilyPrefix:(NSString *)prefix
+                              andManager:(EXFontManager *)manager;
```

Comment:
```suggestion
                              manager:(EXFontManager *)manager;
```

No "and" in Cocoa (this matters especially with Swift's auto-conversion rules)